### PR TITLE
add new version for ms2snoop

### PR DESCRIPTION
**Pull request by Lain-inrae**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

new in version 2.1.0
===

  * Adds support for sirius to match fragments with their formula
  * Adds support for PDF generation from galaxy
  * Fix: default RA threshold, that was too high (0.5 -> 0.05)
  * Fix: default mz decimal number (0 -> 3)
  * Fix: mz decimal: rounded values were used instead of original one in some computing ; rounded values are still output.
  * Fix: missing session info is now display when launched from galaxy.

### CODE REVIEW

**Review by fgiacomoni — APPROVED** [review]
Do you want to leave your manual_test.sh in tool. Please add Kevin Wagner - INRAE as authors with Jean-Francois Martin.

File: tools/ms2snoop/.conda-lock.yml [hunk 1]
```diff
@@ -0,0 +1,1383 @@
+# This lock file was generated by conda-lock (https://github.com/conda-incubator/conda-lock). DO NOT EDIT!
```
Comment (bgruening) [inline tools/ms2snoop/.conda-lock.yml:1]: this file should not be here, isn't it?

Comment (Lain-inrae) [reply tools/ms2snoop/.conda-lock.yml:1]: Thanks, I will remove it!

Comment (Lain-inrae) [reply tools/ms2snoop/.conda-lock.yml:1]: What do you think about the environment.yml file..?

And also, for reproducible deploy (and not have to resolve a tool's dependencies) - how should I provide the conda lock file?

Thanks!

Comment (bgruening) [reply tools/ms2snoop/.conda-lock.yml:1]: environment.yml files seem to be correct, but should also not end up in the TS I guess. For really 99% reproducible deploys we use containers. So there is one container created for your tool already that has all your dependencies. A frozen container for your environment.yml file.

Comment (Lain-inrae) [reply tools/ms2snoop/.conda-lock.yml:1]: I understand, thanks!
I'll fix that.

**Pull request merged**